miscellaneous/build_templates/analysis.yml

```diff
@@ -3,7 +3,7 @@ parameters:
   skipPolaris: false # For samples that have no code Polaris is capable of scanning
   useRequirementsTxt: false # Tell the task to use Python requirements.txt for BlackDuck
   projectPath: '' # (Optional) Tell the task to use a specific path to the project
-  snippetMatching: SNIPPET_MATCHING
+  snippetMatching: 'SNIPPET_MATCHING'
```

Comment:
I don't think this part is necessary. Also, I'm not sure if you've done this but you can branch a sample, change its pipeline to point to this branch, and test it that way.

Reply:
I tested that it works now!